From e0923d27a63b8de1c0274f634acefbc4a7bcf8af Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <panto@antoniou-consulting.com>
Date: Tue, 15 Jan 2013 12:05:11 +0200
Subject: [PATCH 052/177] OF: Overlay: Remove excessive debugging crud

There's no need for such excessive debugging crud anymore. Remove it.

Signed-off-by: Pantelis Antoniou <panto@antoniou-consulting.com>
---
 drivers/of/overlay.c | 41 -----------------------------------------
 1 file changed, 41 deletions(-)

diff --git a/drivers/of/overlay.c b/drivers/of/overlay.c
index f65c3a3..fc7bf5a 100644
--- a/drivers/of/overlay.c
+++ b/drivers/of/overlay.c
@@ -257,9 +257,6 @@ static void of_overlay_device_entry_entry_add(struct of_overlay_info *ovinfo,
 static int of_overlay_notify(struct notifier_block *nb,
 				unsigned long action, void *arg)
 {
-#ifdef DEBUG
-	char *propstr = NULL, *spropstr = NULL;
-#endif
 	struct of_overlay_info *ovinfo;
 	struct device_node *node;
 	struct property *prop, *sprop, *cprop;
@@ -278,9 +275,6 @@ static int of_overlay_notify(struct notifier_block *nb,
 		if (node == NULL)
 			return notifier_from_errno(-EINVAL);
 		prop = NULL;
-#ifdef DEBUG
-		propstr = NULL;
-#endif
 		break;
 	case OF_RECONFIG_ADD_PROPERTY:
 	case OF_RECONFIG_REMOVE_PROPERTY:
@@ -294,9 +288,6 @@ static int of_overlay_notify(struct notifier_block *nb,
 		prop = pr->prop;
 		if (prop == NULL)
 			return notifier_from_errno(-EINVAL);
-#ifdef DEBUG
-		propstr = __of_dump_prop(prop);
-#endif
 		break;
 	default:
 		return notifier_from_errno(0);
@@ -307,38 +298,6 @@ static int of_overlay_notify(struct notifier_block *nb,
 	if (err != 0)
 		return notifier_from_errno(err);
 
-#ifdef DEBUG
-	switch (action) {
-	case OF_RECONFIG_ATTACH_NODE:
-		pr_debug("ATTACH_NODE: %s\n", node->full_name);
-		break;
-	case OF_RECONFIG_DETACH_NODE:
-		pr_debug("DETACH_NODE: %s\n", node->full_name);
-		break;
-	case OF_RECONFIG_ADD_PROPERTY:
-		pr_debug("ADD_PROP:    %s %s%s\n", node->full_name,
-				prop->name, propstr);
-		break;
-	case OF_RECONFIG_REMOVE_PROPERTY:
-		pr_debug("REMOVE_PROP: %s %s%s\n", node->full_name,
-				prop->name, propstr);
-		break;
-	case OF_RECONFIG_UPDATE_PROPERTY:
-		sprop = of_find_property(node, prop->name, NULL);
-		if (sprop)
-			spropstr = __of_dump_prop(sprop);
-		pr_debug("UPDATE_PROP: %s '%s%s' -> '%s%s'\n", node->full_name,
-				prop->name, spropstr,
-				prop->name, propstr);
-		break;
-
-	}
-
-	/* NULL is fine */
-	kfree(propstr);
-	kfree(spropstr);
-#endif
-
 	/* come up with the device entry (if any) */
 	pdev = NULL;
 	state = 0;
-- 
1.8.1.4

